### PR TITLE
fix: Change conferencing app-store URL on event types and bookings pages

### LINKED ISSUE #15811: [CAL-4063] Incorrect app store url in event type page
### Issue Summary
The link to the app store in event type page point to `/apps/categories/video` instead of `apps/categories/conferencing`

### Steps to Reproduce

1. Go to event page 
2. Scroll down to location section & click on App store link

Screenshot 2024-07-17 at 8 29 41 PM

<sub>[CAL-4063](https://linear.app/calcom/issue/CAL-4063/incorrect-app-store-url-in-event-type-page)</sub>

### PR DESCRIPTION
**Pull request by Souptik2001**
## What does this PR do?

- Fixes #15811
- Fixes CAL-4063

/claim #15811

Screenshot 2024-07-23 at 6 03 14 PM

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox.
- [ ] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Go to any event edit page, e.g - `http://localhost:3000/event-types/<event_id>?tabName=setup`
- Scroll to the "Location" section.
- In the text `Can't find the right video app? Visit our App Store`, the "App store" link should be `apps/categories/conferencing`.

### CODE REVIEW

File: apps/web/components/eventtype/Locations.tsx [hunk 1]
```diff
@@ -433,7 +433,7 @@ const Locations: React.FC<LocationsProps> = ({
         <p className="text-default mt-2 text-sm">
           <Trans i18nKey="cant_find_the_right_video_app_visit_our_app_store">
```
Comment (Souptik2001) [inline apps/web/components/eventtype/Locations.tsx:434]: FYI - The text i.e - `cant_find_the_right_video_app_visit_our_app_store` still says "Can't find the right video app". So, please let me know if this has to be also changed to something else.

Comment (anikdhabal) [reply apps/web/components/eventtype/Locations.tsx:434]: Let's make it to conferencing, when everything is conferencing instead of video

**Review by anikdhabal — CHANGES_REQUESTED** [review]

**Review by anikdhabal — CHANGES_REQUESTED** [review]
@Souptik2001 Let's change the translation key and value to `conferencing`, when everything is conferencing instead of video. So, the text would be " Can't find the right conferencing app".

**Comment by Souptik2001** [comment]
@anikdhabal Done!

Also changed the key from `cant_find_the_right_video_app_visit_our_app_store` to `cant_find_the_right_conferencing_app_visit_our_app_store`.

**Important note:** All translations needs to be updated. ⚠️

**Review by anikdhabal — COMMENTED** [review]
Hey @Souptik2001 you don't need to update it in every locale; just change it to /locales/en and rest remove. We handle the rest externally.

**Comment by Souptik2001** [comment]
@anikdhabal Yes that's already done! Thanks! 😄

**Comment by anikdhabal** [comment]
> @anikdhabal Yes that's already done! Thanks! 😄

But I've seen changes made to every locale; only change it to `locales/en`

**Comment by anikdhabal** [comment]
> Just confirming once before reverting.

 Just change the key and text in the `locales/en` and rest remove both the key and text

**Review by PeerRich — DISMISSED** [review]

**Review by anikdhabal — CHANGES_REQUESTED** [review]
@Souptik2001 need to remove the key and text for the other locales; for `locales/en`, you already updated.

**Comment by Souptik2001** [comment]
@anikdhabal Done!

cc: @PeerRich

**Comment by anikdhabal** [comment]
> @anikdhabal Done!
> 
> cc: @PeerRich

And what about removing both the key and text in all locales except `locales/en`? Please do it 😄

**Comment by Souptik2001** [comment]
@anikdhabal I meant I did that only, in the above commit. Am I still missing something? 😓

**Comment by anikdhabal** [comment]
> @anikdhabal I meant I did that only, in the above commit. Am I still missing something? 😅

Nothing just remove both the key and text in all locales except locales/en 😄

**Comment by Souptik2001** [comment]
@anikdhabal I mean it's done, can you please check once. 😅 

Screenshot 2024-07-23 at 7 06 25 PM

**Comment by Souptik2001** [comment]
@anikdhabal Ahh oops.. 😅 got it now!

I was thinking that those will be only modified when updating translations. Sorry for the back-and-forth.

**Comment by anikdhabal** [comment]
> @anikdhabal Ahh oops.. 😅 got it now!
> 
> I was thinking that those will be only modified when updating translations. Sorry for the back-and-forth.

All good, yeah i understood 😅

**Review by anikdhabal — APPROVED** [review]
LGTM!
Approving on behalf of @PeerRich and me

**Comment by suyash5053** [comment]
@anikdhabal Sorry, but the changes done here, are not right. 
@Souptik2001 There was no need to change the i8 tags, you just had to change the URL of the App Store. Else thanks for the time and hard work, looking forward to more of your work. 


cc- @PeerRich

**Comment by Souptik2001** [comment]
@suyash5053 FYI the changes were purposeful and made after confirmation - https://github.com/calcom/cal.com/pull/15878#pullrequestreview-2193892157

Anyhow it was just a matter of a revert commit if the text change was not required..🙂 But ok no worries. 👍

**Comment by anikdhabal** [comment]
@suyash5053, in our app, most places use 'conferencing' to describe the conferencing app, and we have a 'conferencing' section rather than 'video' for installing the conferencing app in the app store, with the link also redirecting there. Don't you think it makes sense to change it to 'conferencing'? I suggest @Souptik2001 do that. I hope you understand.

**Comment by suyash5053** [comment]
@Souptik2001 Ohk, wasn't aware of the fact, No issues, Sorry for the inconvenience casued, if any.
@anikdhabal Yuppp I understand, thanks for explaining.

**Pull request merged**